client/ayon_houdini/plugins/publish/validate_frame_token.py

```diff
@@ -42,11 +43,11 @@ def get_invalid(cls, instance):
         # Check trange parm, 0 means Render Current Frame
         frame_range = node.evalParm("trange")
         if frame_range == 0:
-            return []
+            return
 
         output_parm = lib.get_output_parameter(node)
         unexpanded_str = output_parm.unexpandedString()
 
         if "$F" not in unexpanded_str:
             cls.log.error("No frame token found in '%s'" % node.path())
-            return [instance]
+            return [node]
```

Comment:
This seems like a code change - it may fix a bug of course, but it doesn't sound very related to the PR title?

Reply:
Thank you. it should be done in another PR.

Reply:
https://github.com/ynput/ayon-houdini/pull/193/commits/ed447df6f63e8a3912b53487a679fc0205e0117d